po/common/en.po

```diff
@@ -4834,4 +4834,12 @@ msgstr "Recycling instructions and/or packaging information text has been extrac
 
 msgctxt "product_js_extracting_packaging"
 msgid "Extracting recycling instructions and/or packaging information"
-msgstr "Extracting recycling instructions and/or packaging information"
\ No newline at end of file
+msgstr "Extracting recycling instructions and/or packaging information"
+
+msgctxt "contains"
+msgid "Contains"
```

Comment:
This is a French centric approach. We should have "Contains %" so that translators can potentially do "% Bla Bla"

Reply:
It's to be used as "Contains: XYZ". The issue with "Contains %" is that it will create bad sentences in some languages. e.g. in French, we won't have the right article. "Contient gluten" and "Ne contient pas gluten" sounds bad.

Reply:
"Contains: %" then.

Reply:
Arabic is RTL, so it should be one sentence, otherwise we will have issues.

Reply:
ok

Reply:
@teolemon : fixed, thank you!